src/feedlistformaction.cpp

```diff
@@ -911,6 +904,13 @@ void FeedListFormAction::update_form_title(unsigned int width)
 	std::string title_format =
 		cfg->get_configvalue("feedlist-title-format");
 
+	const unsigned int unread_feeds = std::count_if(
```

Comment:
Considering this is repeated in two places already, why not move this code into a method?

Reply:
Good idea, done